src/services/system/model.js

```diff
@@ -29,6 +29,13 @@ const systemSchema = new Schema(
 		alias: { type: String },
 		oaClientId: { type: String }, // just for oauth2-systems
 		oaClientSecret: { type: String }, // just for oauth2-systems
+		oauthconfig: {
+			client_id: { type: String, required: true },
```

Comment:
is this the same as oaClientId ?

Reply:
as agreed in personal conversation, this is a matter for later.